surrogates/common_utils/fits.py

```diff
@@ -0,0 +1,139 @@
+##==============================================================================
+## BHPTNRSurrogate module
+## Description : evaluates the (spline or GPR) surrogate model
+## Author : Tousif Islam, Nov 2022 [tislam@umassd.edu / tousifislam24@gmail.com]
+##==============================================================================
+
+import numpy as np
+import scipy
+from scipy.interpolate import splrep, splev
+from . import utils
+from .eval_pysur import evaluate_fit as evaluate_GPR
+
+#----------------------------------------------------------------------------------------------------
+def evaluate_GPR_at_EIM_nodes(X, fit_data):
+    """ Evaluate the spline at one EIM node """
+    return None
```

Comment:
use "raise NotImplementedError" instead of "return None"

Reply:
Done!